botocore/utils.py

```diff
@@ -524,8 +525,9 @@ class ArgumentGenerator(object):
         print("Sample input for dynamodb.CreateTable: %s" % sample_input)
 
     """
-    def __init__(self):
-        pass
+    def __init__(self, use_member_names=False):
+        self._use_member_names = use_member_names
+        self._max_stack_depth = 1
```

Comment:
No I'll remove.  Initially I was going to use it to allow you to specify how many stack levels to recurse down for recursive shapes, but it ended up not being needed so I removed it.